src/Resources/contao/templates/modules/mod_login.html5

```diff
@@ -13,16 +13,19 @@
       <?php endif; ?>
       <input type="hidden" name="FORM_SUBMIT" value="<?= $this->formId ?>">
       <input type="hidden" name="REQUEST_TOKEN" value="{{request_token}}">
+      <?php if ($this->targetPath): ?>
+        <input type="hidden" name="<?= $this->targetName ?>" value="<?= $this->targetPath ?>">
```

Comment:
I also wonder if this is prone to manipulation? @ausi /cc

Reply:
Maybe we should check, if it's an existing page from the installation?

Reply:
I don't know. It seems it is a hidden field in Symfony, too: https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/SecurityBundle/Tests/Functional/Bundle/CsrfFormLoginBundle/Form/UserLoginType.php#L47

Reply:
well you can only manipulate it through the web inspector. Which means the redirect target after login is manipulated. Can't really see an issue in that, it does not send any data to that URL.

Reply:
`htmlspecialchars()` is missing for `$this->targetPath`, either in the template or in `ModuleLogin`. As it is now it’s vulnerable to XSS.

Reply:
Fixed in 975070ea3adeb6f202ef55bdf280654a6663a353.